src/cljs/rems/catalogue.cljs

```diff
@@ -124,10 +108,17 @@
 
 (defn catalogue-page []
   (let [language @(rf/subscribe [:language])
-        catalogue @(rf/subscribe [::catalogue])
         loading-catalogue? @(rf/subscribe [::loading-catalogue?])
         drafts @(rf/subscribe [::draft-applications])
-        loading-drafts? @(rf/subscribe [::loading-drafts?])]
+        loading-drafts? @(rf/subscribe [::loading-drafts?])
+        table-spec {:id :catalogue
+                    :columns [{:key :name
+                               :title (text :t.catalogue/header)
+                               :sortable? true
```

Comment:
Shouldn't the default be filterable and sortable?

Reply:
It will be figured out later once this table is used in more places and the duplication is more obvious.